Multisig/UI/App/RemoteNotificationHandler.swift

```diff
@@ -156,6 +180,27 @@ class RemoteNotificationHandler {
         }
     }
 
+    // MARK: - Registering in the service
```

Comment:
In my PR I add the implementation for all those functions, after the merge you can remove them

Reply:
ok